PhysicsPlatformer/Program.cs

```diff
@@ -0,0 +1,134 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using ConsoleGameLib.CoreTypes;
+using ConsoleGameLib.PhysicsTypes;
+using ConsoleGameLib;
+using System.Threading;
+
+
+namespace PhysicsPlatformer
+{
+    class Program
+    {
+
+        static void Main(string[] args)
+        {
```

Comment:
Please comment your _example_ code extensively so it is easy to tell what it does without referring to other code.